.github/workflows/e2e.yaml

```diff
@@ -123,13 +135,12 @@ jobs:
           if [[ inputs.suite == 'PrivateCluster' ]]; then
             TEST_SUITE="Integration"
           fi
-          aws eks update-kubeconfig --name ${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}
-          TEST_SUITE="$TEST_SUITE" ENABLE_METRICS=${{ inputs.enable_metrics }} METRICS_REGION=${{ vars.TIMESTREAM_REGION }} GIT_REF="$(git rev-parse HEAD)" \
-            CLUSTER_NAME="${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}" CLUSTER_ENDPOINT="$(aws eks describe-cluster --name ${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }} --query "cluster.endpoint" --output text)" \
-            INTERRUPTION_QUEUE="${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}" make e2etests
+          aws eks update-kubeconfig --name "${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}"
+          INTERRUPTION_QUEUE="${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}" CLUSTER_NAME="${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}" TEST_SUITE="$TEST_SUITE" ENABLE_METRICS=${{ inputs.enable_metrics }} METRICS_REGION=${{ vars.TIMESTREAM_REGION }} GIT_REF="$(git rev-parse HEAD)" \
+            CLUSTER_ENDPOINT="$(aws eks describe-cluster --name "${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}" --query "cluster.endpoint" --output text)"  make e2etests
       - name: notify slack of success or failure
         uses: ./.github/actions/e2e/slack/notify
-        if: (success() || failure()) && github.event_name != 'workflow_run' && inputs.workflow_trigger != 'versionCompatibility'
+        if: (success() || failure()) && github.event_name != 'workflow_run' && inputs.workflow_trigger != 'versionCompatibility' && inputs.workflow_trigger != 'soak'
```

Comment:
We said that we wanted to notify on slack for the soak testing to start. We just need to designate a separate channel to get the notifications. Should we specify and add that here?